src/main/java/frc/robot/subsystems/Intake.java

```diff
@@ -59,8 +60,15 @@ public void setNeutralMode() {
     rightCenteringMotor.set(0);
   }
 
+  private double getRollerVelocity() {
+    return rollerMotor.getVelocity().getValueAsDouble();
```

Comment:
Since we are requesting a percent output to control it. We should get percent output to read out to smart dashboard

Reply:
nuh uh